docs/serverStatusView.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Server Status View
+nav_order: 3
+---
+
+
+### Server Status View
+
+There is a "MarkLogic Server Explorer" icon ( 
```

Comment:
I try to avoid passive sentences like "There is" - try an active voice like "mlxprs includes an icon (or button? Not sure of the right term) in the VSCode activity bar, which VSCode displays by default on the left side of the VSCode window. When selected...".